VocaDbModel/Domain/Globalization/RegionCollection.cs

```diff
@@ -8,27 +6,23 @@ namespace VocaDb.Model.Domain.Globalization
 {
 	public class RegionCollection
 	{
-		// FIXME
-		public static readonly string[] RegionCodes = CultureInfo.GetCultures(CultureTypes.SpecificCultures)
-			.Select(culture => new RegionInfo(culture.Name).TwoLetterISORegionName)
-			.OrderBy(c => c)
-			.Distinct()
+		public static readonly IEnumerable<RegionInfo> DefaultRegions = CultureInfo.GetCultures(CultureTypes.SpecificCultures)
+			.Select(culture => new RegionInfo(culture.Name))
+			.GroupBy(region => region.TwoLetterISORegionName)

```

Comment:
We also have Distinct with property selector

Reply:
Oh, didn't know that, thanks!